core/vm/cvm.go

```diff
@@ -0,0 +1,42 @@
+package vm
+
+import (
+	"github.com/ledgerwatch/erigon/common"
+)
+
+func NewCVM(state IntraBlockState) *CVM {
+	cvm := &CVM{
+		intraBlockState: state,
+	}
+
+	return cvm
+}
+
+type CVM struct {
+	config          Config
+	intraBlockState IntraBlockState
+}
+
+func (cvm *CVM) Create(code []byte) ([]byte, common.Address, error) {
+	ret, err := cvm.run(code)
```

Comment:
I think Cairo constructors are different, they don't just return bytecode. 

So,

(1) no need to run anything here
(2) we can store `code` instead of `ret`.

later we will have to run a constructuctor (could be a TODO), but that only after integration with CairoOS and state because these are the parts that enforce those limits